pubsub-service-api.yml

```diff
@@ -413,65 +409,58 @@ components:
           properties:
             reason:
               type: string
-              description: Mandatory string identifying the type of error
+              description: "A mandatory code identifying the error type."
```

Comment:
if you say "code", I expect a number (as in HTTP status code), but that's not the case here I believe